Makefile

```diff
@@ -1,11 +1,8 @@
 build:
-	docker build -t stackstorm/stackstorm:latest st2-upstart
+	docker build -t stackstorm/stackstorm:latest images/eval
```

Comment:
`eval` is a bad name for the image. It conveys our speculation on one of potentially multiple uses. WE ARE DONE with this image, it will be used. Call it `st2`, and when you have others you can do `st2-what-so-special-about-me`. 

Names always give me trouble (or the other way around :)